app.js

```diff
@@ -80,14 +81,16 @@ app.get('/wetty/ssh/:user', function(req, res) {
 });
 app.use('/', express.static(path.join(__dirname, 'public')));
 
+var port = process.env.NODE_ENV == 'production' ? 'systemd' : opts.port;
```

Comment:
Why not putting 'systemd' in opts.port directly when we need it? This value ends up in a `port` variable anyway, right? (I'm ok with that, it's a broader meaning for 'port' than just port number).
Also I feel like:
- it is orthogonal from the NODE_ENV variable meaning, 
- it would minimize this patch.